post_installation_configuration/cluster-tasks.adoc

```diff
@@ -162,6 +162,47 @@ xref:../installing/installing_aws/installing-aws-network-customizations.adoc#ins
 
 include::modules/images-update-global-pull-secret.adoc[leveloffset=+1]
 
+[id="adding-worker-nodes_{context}"]
+== Adding worker nodes
+
+After you deploy your OpenShift Container Platform cluster, you can add worker nodes to scale cluster resources. There are different ways you can add worker nodes depending on the installation method and the environment of your cluster.
+
+=== Adding worker nodes to IPI clusters
+
+For installer-provisioned infrastructure (IPI) clusters, you can manually or automatically scale the machineset to match the number of available bare-metal hosts.
+
+To add a bare-metal host, you must configure all network prerequisites, configure an associated `baremetalhost` object, then provision the worker node to the cluster. You can add a bare-metal host manually or by using the web console. 
+ 
+* xref:../scalability_and_performance/managing-bare-metal-hosts.adoc#adding-bare-metal-host-to-cluster-using-web-console_managing-bare-metal-hosts[Adding worker nodes using the web console] 
+
+* xref:../scalability_and_performance/managing-bare-metal-hosts.adoc#adding-bare-metal-host-to-cluster-using-yaml_managing-bare-metal-hosts[Adding worker nodes using YAML in the web console]
+
+* xref:../installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc#preparing-the-bare-metal-node_ipi-install-expanding[Manually adding a worker node to an IPI cluster]
+
+=== Adding worker nodes to UPI clusters
+
+For user-provisioned infrastructure (UPI) clusters, you can add worker nodes using RHEL or RHCOS ISO images and connect it to your cluster using cluster ignition configs. For RHEL worker nodes, the following example uses ansible playbooks to add worker nodes to the cluster. For RHCOS worker nodes, the following example uses an ISO image and network booting to add worker nodes to the cluster. 
+
+* xref:../post_installation_configuration/node-tasks.adoc#post-install-config-adding-fcos-compute[Adding RHCOS worker nodes to a UPI cluster]
+
+* xref:../post_installation_configuration/node-tasks.adoc#post-install-config-adding-rhel-compute[Adding RHEL worker nodes to a UPI cluster]
+
+=== Adding worker nodes to AI clusters
```

Comment:
Maybe "Adding worker nodes to clusters managed by the assisted installer" would be clearer than using the acronym AI here?